src/index.html

```diff
@@ -8,7 +8,42 @@
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="page-top">
+    <header class="header">
+      <a href="#page-top"><img src="images/logo.png" alt="logo"></a>
```

Comment:
```suggestion
      <a href="#page-top">
        <img src="images/logo.png" alt="logo">
      </a>
```
Consider fixing indentations according to the code style guide